src/📖.jl

```diff
@@ -1,17 +1,31 @@
-## Contains mapping from functions to func_to_emojis
+## Contains mapping from functions to 😃📖
 ## Each function (treated as a Symbol) maps to a Tuple of Symbols and Tuple{Symbol,Float64} for
 ## emojis needing a specific version
-const func_to_emojis = 📖(
+const 😃📖 = 📖(
     ## Base
     :ENV => (:(🧧),),
+    # Symbol(Char(0x1fae5))
     :ArgumentError => (:(💬🚨),),
+    :AbstractChar => ((Symbol(Char(0x1fae5) * '🚗'), 1.8),),
+    :AbstractDict => ((Symbol(Char(0x1fae5) * '📖'), 1.8),),
+    :AbstractDisplay => ((Symbol(Char(0x1fae5) * '📺'), 1.8),),
+    :AbstractFloat => ((Symbol(Char(0x1fae5) * Char(0x1f6df)), 1.8),),
+    :AbstractString => ((Symbol(Char(0x1fae5) * '🧵'), 1.8),),
+    :Bool => (:(👍👎),),
+    :Char => (:(🚗),),
     :Dict => (:(📖),),
-    :Threads => (:(🧵),),
+    :IO => ((Symbol(Char(0x1fa80) * '½'), 1.2), :(👁️😲),), # 🪀½
+    :Threads => ((Symbol(Char(0x1faa2)), 1.5),),
+    :Tuple => (:(👯),),
```

Comment:
I got some feedback that this looks more like `Pair`

Reply:
This could be fixed by adding `🍐`!

Reply:
I will just remove `Tuple` to get this merged. Let's add it pack in another PR.